src/services/storage/s3.js

```diff
@@ -1,6 +1,8 @@
-import dayjs from 'dayjs';
-import { Endpoint, S3 } from 'aws-sdk';
+import { NoS3ObjectsFoundError, S3ObjectDeletionError } from '../../errors/errors';
+import { logInfo } from '../../middleware/logging';
 import { initializeConfig } from '../../config';
+import { Endpoint, S3 } from 'aws-sdk';
+import dayjs from 'dayjs';
```

Comment:
Is this import used? I might be going blank

Reply:
Yes, being used on line 29!

Reply:
Yes, all imports are being used!